src/components/showcase-settings.component.tsx

```diff
@@ -54,12 +59,13 @@ export const ShowcaseSettings = (props: ShowcaseSettingsProps): React.ReactEleme
     setSettingsMenuVisible(false);
   };
 
-  const createThemesMenuItems = (): OverflowMenuItemType[] => {
+  const createThemesMenuItems = (): any => {
     return props.themes && props.themes.map(createThemeMenuItem);
   };
 
-  const createSettingsMenuItems = (): OverflowMenuItemType[] => {
-    const settings = props.settings && props.settings.map(createSettingMenuItem);
+  const createSettingsMenuItems = (): any => {
+    const settings =
+      props.settings && props.settings.map(createSettingMenuItem);
     return settings || [];
   };
```

Comment:
It's better to not use `any` where possible.
`React.ReactElement` is a generic type and covers most of the cases. 

```suggestion
  const createSettingsMenuItems = (): React.ReactElement[] => {
    const settings = props.settings && props.settings.map(createSettingMenuItem);
    return settings || [];
  };
```